documentation/basics/headless.md

```diff
@@ -89,75 +88,72 @@ the command (as shown in the example above).
-  Please always specify the values in the _5D_ order `(t,c,z,y,x)`.
-  Use `None` for dimensions that are not present in your data.
+- `--cutout_subregion` Subregion from the original to imput to operate on. The expected format is `[(axis_1_start,axis_2_start,...,axis_n_start),(axis_1_stop,axis_2_stop,...,axis_n_stop)]`. The order of the axis is the same order as in your input image. The start values are inclusive and the stop values are exclusive. You can replace any value with `None` to have it be interpreted as `0` in the start values or the maximum value in that axis for the stop values. If you specify a region that exceeds the shape of your input image, ilastik will crop your selection so that it fits the input shape.
+    E.g.: If your image is a png file with axis x y c, then you could run a headless prediction on a 50x50 square starting from 10x10 and containing all channels by specifying `--cutout_subregion="[(10,10,None),(60,60,None)]"`.
 - `--export_dtype` The pixel type to convert your results to.  Choices are: `uint8`, `uint16`, `uint32`, `int8`, `int16`, `int32`, `float32`, `float64`.  Note that some formats don't support every pixel type.
 - `--output_axis_order` Transpose the storage order of the results. For example, this affects the sliced dimension for stack outputs.
 - `--pipeline_result_drange` Pipeline result data range (min,max) BEFORE normalization, e.g. `"(0.0,1.0)"`
 - `--export_drange` Exported data range (min,max) AFTER normalization, e.g. `"(0,255)"`
 
-## Headless Mode for Object Classification
+[Data Export Applet]: {{site.baseurl}}/documentation/basics/export.html#data-export-applet-ss
 
-The [Object Classification Workflow] can also be used in "headless" mode.  
-The command-line interface is similar to the Pixel Classification interface described above, but with a few minor changes.  
-Here's an example command (explanation below):
+## Headless Mode for Pixel Classification
 
-[Object Classification Workflow]: {{site.baseurl}}/documentation/objects/objects.html
+When running the Pixel Classification Workflow in headless mode, the available options for the `--export_source` flag are:
 
-    $ ./run_ilastik.sh --headless
-                       --project=MyProject.ilp
-                       --table_filename=/tmp/exported_object_features.csv
-                       --export_source="Object Predictions"
-                       --raw_data "my_grayscale_stack_1/*.png"
-                       --segmentation_image my_unclassified_objects_1.h5/binary_segmentation_volume
+- `"Probabilities"`, which exports a multi-channel image where pixel values represent the probability that that pixel belongs to the label represented by that channel;
+- `"Simple Segmentation"`, which produces an image where pixel values indicate the class to which a pixel belongs. For this image, every pixel with the same value should belong to the same class of pixels;
```

Comment:
```suggestion
- `"Simple Segmentation"`, which produces a single channel image of integer pixel values, where pixel values indicate the class to which a pixel belongs. For this image, every pixel with the same value should belong to the same class of pixels;
```

Reply:
okay I see you use the term "image" synonymous for "single channel image" and indicate where images are multichannel. That's also good.

Reply:
"Explicit is better than implicit". Fixing it to single-channel